build.js

```diff
@@ -0,0 +1,23 @@
+const ClosureCompiler = require('google-closure-compiler').compiler;
+
+const closureCompiler = new ClosureCompiler({
+    js: './src/*.js',
+    entry_point: './src/index.js',
+    js_output_file: './proxy.min.js',
+    language_in: 'ECMASCRIPT6',
+    language_out: 'ECMASCRIPT5',
+    compilation_level: 'ADVANCED_OPTIMIZATIONS',
+    dependency_mode: 'STRICT',
+    process_common_js_modules: true,
+    output_wrapper: '(function(){%output%})()', // this prevents closure compiler from polluting the global scope
+    jscomp_off: 'undefinedVars' // `global` and `process` are not defined...
```

Comment:
We can probably fix this by providing externs (it's just for environment detection) or another hack. Fine for now though.